get-started/go-live.md

```diff
@@ -1,15 +1,20 @@
 ---
-layout: post
+layout: page
 title: Going Live with Search.gov
 crumbname: Go-Live
 category: manual
-redirect_from: "/blog/go-live.html"
 tags: help-manual
+date: September 4, 2019
+last_modified_at: September 4, 2019
 breadcrumbs:
```

Comment:
We should either update the Manual item in these breadcrumbs, or remove the breadcrumbs